test/box/cfg.result

```diff
@@ -284,7 +284,7 @@ box.cfg{coredump = 'true'}
  | - error: 'Incorrect value for option ''coredump'': should be of type boolean'
  | ...
 
--- check comment to issue #2191 - bad argument #2 to ''uri_parse''
+-- check comment to issue #2191 - bad argument #2 to ''uri_create''
```

Comment:
Please revert this part. It isn't needed.

Reply:
fixed